cpu/stm32_common/cpu_common.c

```diff
@@ -20,20 +20,65 @@
 
 #include "periph_cpu_common.h"
 
-void periph_clk_en(uint8_t bus, uint32_t mask)
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+void periph_clk_en(bus_t bus, uint32_t mask)
 {
     if (bus == APB1) {
         RCC->APB1ENR |= mask;
-    } else {
+    }
+    else if (bus == APB2) {
```

Comment:
hm, now looking at this again, it seems this is exactly one of these situations that `switch` was made for, right?!